mlir/docs/Tutorials/DataFlowAnalysis.md

```diff
@@ -154,7 +146,7 @@ shown below:
 /// This class represents a lattice element holding a specific value of type
 /// `ValueT`.
 template <typename ValueT>
-class LatticeElement ... {
+class Lattice ... {
```

Comment:
The comment says `This class represents a lattice element...` , why did you rename it?